shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
The way the code is now, we only fetch one line of chat. We probably still don't want to fetch this one line if it has already been displayed. I guess the right thing to do is probably to store the most recent timestamp and fetch only stuff that has been added after that.